src/Backend/Modules/Blog/Tests/Actions/EditTest.php

```diff
@@ -28,11 +28,7 @@ public function testWeCanGoToEditFromTheIndexPage(Client $client): void
         $this->login($client);
 
         $this->assertPageLoadedCorrectly($client, '/private/en/blog/index', [LoadBlogPosts::BLOG_POST_TITLE]);
-
-        $link = $client->getCrawler()->selectLink(LoadBlogPosts::BLOG_POST_TITLE)->link();
-        $client->click($link);
-
-        $this->assertIs200($client);
+        $this->assertClickOnLink($client, LoadBlogPosts::BLOG_POST_TITLE, [LoadBlogPosts::BLOG_POST_TITLE]);
         $this->assertCurrentUrlContains($client, '&id=1');
```

Comment:
Use a constant for the id like the other tests

Reply:
Fixed in ec7b66c48